### DISCUSSION TITLE
Safe Exam Browser 3.8.0 - Installs but doesn't launch

### DISCUSSION THREAD

**Discussion by UOS-David** [SafeExamBrowser/seb-win-refactoring#1038] [General]
I have packaged Safe Exam Browser 3.x over the last couple of years and have hit a barrier with Safe Exam Browser 3.8.0

The software installs but doesn't launch and just get an error in Event Viewer

3.6 and 3.7 works fine

[seblog.txt](https://github.com/user-attachments/files/17918849/seblog.txt)

![3 8 0-eventviewer-error](https://github.com/user-attachments/assets/b4c74973-4fe2-41d0-a91a-5065d8c61345)

**Comment by dbuechel** [comment]
Are there any log files of SEB itself? If so, please do provide them. Furthermore, please specify which SEB build you're using exactly and on which operating system version.

**Comment by UOS-David** [comment]
Damian

After some investigation we have discovered that the issue may be our
antivirus / malware protection "Crowdstrike"

The puzzling issue is that Crowdstrike doesn't detect it as a treat but It
doesn't load if you install 3.8.0

The application doesn't get to a point where it writes a log file in the
location you mentioned

We are using Windows 11 23H2 Education

David

**Reply by dbuechel** [reply]
Thanks for the important information. You're thus saying that when you uninstall / disable Crowdstrike, everything works as expected?

**Comment by UOS-David** [comment]
Damian

Yes - We uninstalled Crowdstrike and it works fine 

Have you seen any other institutions have this issue with 3.8 and Crowdstrike

We have tried exclusions on the  C:\Program Files\SafeExamBrowser but still no luck 

David

**Comment by dbuechel** [comment]
Thanks for the update. No, up and until today we have not heard from issues with Crowdstrike, albeit that your issue appears to be related to #1023.

If you happen to have the time, it would be great if you could test the latest beta version of SEB 3.9.0 and let us know if the issue with Crowdstrike persists. You can download the build from our build server resp. here directly: https://sebdev.ethz.ch/api/buildjobs/92bl4tf1bk7b2gy6/artifacts/SEB_3.9.0.774_SetupBundle.exe.

**Comment by UOS-David** [comment]
Good News !!!

Crowdstrike have tweaked their detections and Safe Exam Browser 3.8.0 now loads up ok
Crowdstrike said that some programs were attempting to write to protected memory and this caused them to not load 
If you have a falcon sensor greater than 7.19 then it will now load !!

David

**Reply by dbuechel** [reply]
Great, that's excellent news! Thanks for sharing the information.

**Reply by dbuechel** [reply]
Were you able to test the latest beta version of 3.9.0 with an older version of Crowdstrike (where the issue occurred, i.e. SEB didn't start)?
